roles/openshift_common/tasks/main.yml

```diff
@@ -0,0 +1,14 @@
+---
+# fixme: Once openshift stops resolving hostnames for node queries remove this...
+- name: Set hostname to IP Addr (WORKAROUND)
+  hostname: name={{ openshift_bind_ip }}
```

Comment:
That is a very good question, I moved it to openshit_common because it was configured in both openshift_master and openshift_node.

@jwhonce is the hostname workaround needed on the master as well?